src/pages/ProfileRegisterPage.tsx

```diff
@@ -1,3 +1,152 @@
+import { useState } from "react";
+
+import { useNavigate } from "react-router-dom";
+
+import { SeoulDistrict, SeoulDistricts } from "../types";
+
+import { putUser } from "@/apis/services/userService";
+import { Close } from "@/assets/icon";
+import Button from "@/components/Button";
+import Select from "@/components/Select";
+import TextField from "@/components/TextField";
+import { useUserStore } from "@/hooks/useUserStore";
+import { autoHyphenFormatter } from "@/utils/phoneNumber";
+
+type FormType = {
+  name: string;
+  phone: string;
+  address: SeoulDistrict | undefined;
+  bio: string;
+};
+
+const FIELD_LABELS: Record<keyof FormType, string> = {
+  name: "이름",
+  phone: "연락처",
+  address: "선호 지역",
+  bio: "소개",
+};
+
 export default function ProfileRegisterPage() {
-  return <div>ProfileRegisterPage</div>;
+  const navigate = useNavigate();
+  const { user } = useUserStore();
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const [form, setForm] = useState({
+    name: "",
+    phone: "",
+    address: undefined,
+    bio: "",
+  });
+
+  const handleChange = (
+    key: keyof typeof form,
+    value: string | SeoulDistrict,
+  ) => {
+    setForm((prev) => ({ ...prev, [key]: value }));
+  };
+
+  const handleSubmit = async () => {
+    if (!user?.id) {
+      alert("로그인 정보가 없습니다.");
+      return;
+    }
+
+    if (isSubmitting) return;
+
+    const requiredFields: Array<keyof typeof form> = ["name", "phone"];
```

Comment:
```suggestion
    const requiredFields: Array<keyof FormType> = ["name", "phone"];
```
이 변수는 컴포넌트 외부로 꺼내도 좋을 것 같아요. 🤔